File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {
    this.findAll = function () {
        return $http.get("../brand/findAll.do");
    }

    this.findPage=function (pageNum, pageSize) {
        return $http.get("../brand/findPage.do?pageNum=" + pageNum + "&pageSize=" + pageSize)
    }

    this.findOne=function (id) {
        return $http.get("../brand/findOne.do?id=" + id)
    }

    this.add = function (entity) {
        return $http.post("../brand/add.do", entity)
    }

    this.update = function (entity) {
        return $http.post("../brand/update.do",entity)
    }

    this.dele = function (ids) {
        alert(ids)
        return $http.get("../brand/delee.do?ids=" + ids)
        alert(2)
    }

})

File: pinyougou_manager_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import com.pinyougou.entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    List<TbBrand> findAll();

    PageResult findPage(Integer pageNum, Integer pageSize);

    void add(TbBrand tbBrand);

    TbBrand findOne(Long id);

    void update(TbBrand tbBrand);

    void dele(Long[] ids);

    List<Map> SelectBrandList();

}
